app/models/registration.rb

```diff
@@ -67,7 +67,7 @@ class Registration < ApplicationRecord
   end
 
   def update_lanes!(params, acting_user)
-    Registrations::Lanes::Competing.update!(params, self, acting_user.id)
```

Comment:
Why are you removing the `.id` here? The `update!` method does need the ID, not the whole entity

Reply:
Ok so "why did this work" has a few interrelated answers:
- acting_user_id was only used to write the user_id to the registration history
- tests didn't assert the contents of the registration history (I've since added a test to fix this) 
- `update_lanes!` is only used in 2 places in the code since the refactor of lanes/competing.rb: 
  - in tests
  - in this auto accept code

We should update `update_lanes!` invocations to use `update_raw` instead - my suggestion is that this happens in a follow-up PR, given that this PR is already pretty busy? I can do it here too, it'll just increase the diff (particularly in the tests)

Reply:
I'm happy to get rid of `update_lanes!`, and you're welcome to do that in a follow-up PR.

However, please keep in mind that `update_raw` loads a separate copy of the registration from the database and operates on that volatile copy. (This is why we previously had that infamous `reload` that nobody knew how to get rid of). In contrast, `update!` has the advantage of operating directly on an existing `Registration` model (which you can see here with the `self` being passed directly into the method).

So in that follow-up PR, just be careful about which one you're using